hypervisor/start.c

```diff
@@ -141,189 +138,50 @@ void drawing_desktop()
     //    putfont8_string(vram,xsize, 28, 48, COL8_FFFFFF,font.Bitmap , "Memman Free 2 Failed");
     //}
     //sprintf(buf,"MEMORY %d MB. %dMB Heap Free.", memtotal / (1024*1024), memman_total(memman) / (1024*1024));
-    putfont8_string(vram,xsize, 8, 8, COL8_FFFFFF,font.Bitmap , "Hack Week 0x10!!!");
+    putfont8_string(vram,xsize, 8, 8, COL8_FFFFFF,font.Bitmap , "Hack Week 0x13!!!");
```

Comment:
should be 0x12??

Reply:
It's doesn't matter, it is not fixed, I have to modify this on every Hackweek. ;-)

Reply:
Indeed, it's HW18, I counted like 2019)